src/sources/readerthread.cpp

```diff
@@ -521,6 +519,24 @@ bool ReaderThread::checkFormat()
 	  }
 	}
   }
+  else if ( m_format == ReadEvent::DO3122Continuous )
+  {
+      if ( (static_cast<uint8_t>(m_fifo[(m_length-21+FIFO_LENGTH)%FIFO_LENGTH]) != 0xAAu)
+           || (static_cast<uint8_t>(m_fifo[(m_length-20+FIFO_LENGTH)%FIFO_LENGTH]) != 0x55u)
+           || (static_cast<uint8_t>(m_fifo[(m_length-19+FIFO_LENGTH)%FIFO_LENGTH]) != 0x52u)
+           || (static_cast<uint8_t>(m_fifo[(m_length-18+FIFO_LENGTH)%FIFO_LENGTH]) != 0x24u) )
+      {
```

Comment:
You have forgotten this.

Reply:
I hoped you hadn't noticed! Ok, will fix.